tools/build_info.hh

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+#pragma once
+
+#include "build_mode.hh"
+
+namespace tools::build_info {
+enum class build_type { debug = 0, release, dev, sanitize, coverage, unknown };
```

Comment:
Why is `unknown` needed? this variant seems dead, `get_build_type()` will never return it.

Reply:
Removed in the new  version.